stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/3585c6a3fa8e4e0917c0bff6780a3e01cbb1060d/snapshots/cardano-1.21.2.yaml
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/940b9e4bb23fb7c38ce54a7b29174bf49d9390d8/snapshots/cardano-1.24.1.yaml
```

Comment:
I believe we also need to fix `cabal.project`? It appears this project doesn't use stack2cabal, so it probably should be done manually.

Reply:
> This project doesn't use stack2cabal

I usually re-generate the cabal project manually using stack2cabal indeed.

Reply:
One annoying thing though is that the hydra jobset expects all the revision to come with a `--sha256:`  as a comment, and they aren't generated by stack2cabal. I am not sure whether this is a standard cabal thing that should be considered in stack2cabal or if it's simply our CI setup which somehow expects this.

Reply:
(see e.g. https://hydra.iohk.io/jobset/Cardano/cardano-rest-pr-106#tabs-errors)

Reply:
I usually use cardano-repo-tool to get the `--sha256` lines, but really this should be easier. I'll ask on slack...